Apps/WebClient/src/ClientApp/src/store/modules/comment/actions.ts

```diff
@@ -1,56 +1,44 @@
-import { ActionTree, Commit } from "vuex";
+import { ActionTree } from "vuex";
 
 import { ResultType } from "@/constants/resulttype";
-import { Dictionary } from "@/models/baseTypes";
-import RequestResult from "@/models/requestResult";
-import { CommentState, RootState } from "@/models/storeState";
+import { ResultError } from "@/models/requestResult";
+import { CommentState, LoadStatus, RootState } from "@/models/storeState";
 import { UserComment } from "@/models/userComment";
 import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import container from "@/plugins/inversify.config";
 import { ILogger, IUserCommentService } from "@/services/interfaces";
 
 const logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
 
-function handleError(commit: Commit, error: Error) {
-    logger.error(`ERROR: ${JSON.stringify(error)}`);
-    commit("commentError");
-}
-
 const commentService: IUserCommentService = container.get<IUserCommentService>(
     SERVICE_IDENTIFIER.UserCommentService
 );
 
 export const actions: ActionTree<CommentState, RootState> = {
-    retrieveProfileComments(
-        context,
-        params: { hdid: string }
-    ): Promise<RequestResult<Dictionary<UserComment[]>>> {
+    retrieve(context, params: { hdid: string }): Promise<void> {
```

Comment:
retrieve method always returns void doesn't make sense. Why don't we just return the same object regardless if we commit vuex or not? Or come up with a name that means we commit only to the Vuex store.
Also, in some other retrieve of the other services i.e. lab... we don't return void.